contracts/BC_fusion/lib/RLPDecode.sol

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.17;
+
```

Comment:
Can you comment  where is file comes from, and this is unnecessary to be auditted.

Reply:
Done